benchbuild/environments/adapters/buildah.py

```diff
@@ -400,7 +400,7 @@ def _create(self, tag: str, from_: model.FromLayer) -> model.MaybeContainer:
         image = model.Image(tag, from_, [])
 
         # container_id, err = run(bb_buildah('from')[from_.base])
-        res = run(bb_buildah('from')[from_.base])
+        res = run(bb_buildah('from')[from_.base.lower()])
```

Comment:
Wouldn't it make sense to use the existing conversion function `oci_compliant_name()` from `commands.py`?

I can't say for sure whether this is the correct location to apply this transformation since I don't know whether this is the only location where `FromLayer`s are handled.
My first idea would have been to use a converter in the `FromLayer` class as is done with the commands, i.e.,
```
class FromLayer(Layer):
    base: str = attr.ib(converter=oci_compliant_name)
```

Reply:
You are right. For this I just used the same behavior as in other locations in the same module. Using oci_compliant_name is the sane way to go.